src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -280,28 +335,28 @@ sealed class NumericRange[T](
           if ((size & 1) == 0) (size / 2) * (a + b)
           else size * {
             // Sum is even, but we might overflow it, so divide in pieces and add back remainder
-            val ha = a/2
-            val hb = b/2
-            ha + hb + ((a - 2*ha) + (b - 2*hb)) / 2
+            val ha = a / 2
+            val hb = b / 2
+            ha + hb + ((a - 2 * ha) + (b - 2 * hb)) / 2
```

Comment:
Adding spaces around `*` makes it harder to quickly read the precedence of operations.  Having space (correctly) suggest precedence helps rapid understanding.

Reply:
Reverted unintended style changes.